### ISSUE TITLE
When training the model, the memory usage continues to increase until the memory overflows.

### ISSUE THREAD

**Issue by langli1999** [open-mmlab/mmdetection#11515]
**Environment**

1. Python=3.8  pytorch=1.12  cuda=10.2
2. mmcv-full=1.7.2  mmdet=2.28.2

**The command of training**

python tools/train.py configs/exp/faster_rcnn_r50_fpn_1x_mydataset.py --work-dir /xxx/xxx --gpu-id 0 --auto-scale-lr

**Comment by langli1999** [comment]
Except for some hyperparameter adjustments, no changes were made to the model.

**Comment by langli1999** [comment]
Forgot to say one important thing. The usage of video memory does not occur soon after training, and the video memory overflow occurs. Instead, it took a while to train, and in fact, the model went through more than 4,000 iterations before there was a memory overflow. Over the course of more than 4,000 iterations, the memory grew slowly.

**Comment by langli1999** [comment]
Forgot to mention one important thing. Memory usage does not cause memory overflow soon after training. Instead, it was trained for a period of time. In fact, the model went through more than 4,000 iterations before memory overflow occurred. Over the course of more than four thousand iterations, the memory slowly grew.

**Comment by merchen911** [comment]
> Forgot to mention one important thing. Memory usage does not cause memory overflow soon after training. Instead, it was trained for a period of time. In fact, the model went through more than 4,000 iterations before memory overflow occurred. Over the course of more than four thousand iterations, the memory slowly grew.

same problem :( 

My vram was increasing steadily while training the Cascade Mask R-CNN.
